cmd/influxd/config_test.go

```diff
@@ -174,6 +174,7 @@ enabled = true
 
 [logging]
 file   = "influxdb.log"
+write-tracing-enabled = true
```

Comment:
remove enabled. It's in the logging section and it's boolean. `write-tracing = true` is shorter and cleaner. And we're going to have a ton of options here so we'd end up having `enabled` on the end of every one of those.